sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/FlatMapElements.java

```diff
@@ -82,54 +82,45 @@
    */
   public static <InputT, OutputT> FlatMapElements<InputT, OutputT>
```

Comment:
Also, in the interest of standardizing on `into` first, why not make `via` a non static method?